ikarus/controlroutines/pathfollowingtechnique.hh

```diff
@@ -16,54 +18,86 @@
 
 namespace Ikarus {
 
-  struct PathFollowingInformation {
+  struct ControlInformation {
     bool success{false};
+    std::vector<Ikarus::NonLinearSolverInformation> solverInfos{};
+    int totalIterations;
   };
 
-  template <typename NonLinearSolver, typename PathFollowingType = Ikarus::StandardArcLength>
-  requires Concepts::PathFollowingStrategy<PathFollowingType, typename NonLinearSolver::NonLinearOperator>
-  class PathFollowing : public IObservable<ControlMessages> {
+  template <typename NonLinearSolver, typename PathFollowingType = Ikarus::StandardArcLength,
+            typename AdaptiveStepSizing = Ikarus::DefaultAdaptiveStepSizing<>>
```

Comment:
Now the default is too use adaptive step sizing. This is out of sync with the documentation and furthermore i think the default should be not doing it. Therefore there needs to be a default function such as `ikarus/controlroutines/adaptivestepsizing.hh` which is doing nothing

Reply:
Included `NoAdaptiveStepSizing` as per our discussion